runtime/moonbase/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = 'moonbase-runtime'
 description = 'Moonbase Runtime'
 homepage = 'https://moonbeam.network'
 license = 'GPL-3.0-only'
-version = '0.8.4'
+version = '0.9.2'
```

Comment:
We can't simultaneously. follow semantic versioning on the runtime crate, and also keep all the crate version numbers in sync.

I kind of prefer semver for these and thought that's what we decided. But I'm okay if we want to keep crate versions in sync.

I admit I also bumped some crate versions in the recent past that did not need bumped according to semver, because I didn't exactly what the plan was for those.

Reply:
Yes, you are right. So they should have been bumped before this release right? Cause pretty much every change in the runtime would lead to a major version bump no ?